projects/ui/src/lib/services/po-notification/po-notification.service.spec.ts

```diff
@@ -99,5 +103,36 @@ describe('PoNotificationService:', () => {
       notificationService['observableOnClose'](fakeRef);
       expect(spy).toHaveBeenCalled();
     });
+
+    it('must destroy toaster when `stackTop` is more than 5', fakeAsync(() => {
+      for (let i = 0; i < 6; i++) {
+        notificationService.success({
+          message: '',
+          orientation: PoToasterOrientation.Top,
+          action: () => {
+            alert('');
+          },
+          actionLabel: 'close'
+        });
+      }
+      tick(301);
+      expect(notificationService.stackTop.length).toBe(5);
+    }));
+
+    it('must destroy toaster when `stackBottom` is more than 5', fakeAsync(() => {
```

Comment:
Usamos should como padrão.
```suggestion
    it('should destroy toaster when `stackBottom` is more than 5', fakeAsync(() => {
```